src/android/com/silkimen/cordovahttp/CordovaHttpPlugin.java

```diff
@@ -47,6 +47,13 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
 
       this.tlsConfiguration.setHostnameVerifier(null);
       this.tlsConfiguration.setTrustManagers(tmf.getTrustManagers());
+
+      if (this.preferences.contains("androidblacklisttlsprotocols")) {
```

Comment:
I guess accessing preferences isn't case sensitive?

Reply:
Actually it is case-sensitive, but the key is normalized to lower-case string.